src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,21 +157,29 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
-
+    std::optional<data::Blob> ledgerObject;
+    auto output = LedgerEntryHandler::Output{};
+    output.deletedLedgerIndex = std::nullopt;
+    ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
```

Comment:
`auto ledgerObject` here is fine, no need to create an empty optional first just to overwrite it two lines down 👍 
And then let's add a line of space between this and the if.
We also usually don't add curly braces for one line `if`s.